docs/book/v2/advanced.md

```diff
@@ -265,8 +265,8 @@ This now requires a factory to inject the form instance:
 ```php
 namespace Application\Controller;
 
-use Interop\Container\ContainerInterface;
 use Application\Form\MyForm;
+use Psr\Container\ContainerInterface;
```

Comment:
The current pull request adds the support for the PSR container interface for version 3 and not version 2. Therefore this change is wrong.

Reply:
! Thanks for pointing that out, I'll make sure to fix.